app/reducers/views/team.js

```diff
@@ -44,6 +44,29 @@ function lastChannelForTeam(state = {}, action) {
             [action.teamId]: channelIds,
         };
     }
+    case ViewTypes.REMOVE_LAST_CHANNEL_FOR_TEAM: {
+        const {data} = action;
+        const team = state[data.teamId];
```

Comment:
Looks like a convention but I was confused at first about the name `team`. 0/5 but maybe call it `lastChannelIds`?